### PR TITLE
Update bitrise yml for v31.x branch

### PR DESCRIPTION
**Pull request by dnarcese**
I changed the setting on Bitrise and then copied the yml it created - which is why there are a bunch of whitespace changes

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -428,7 +428,7 @@ workflows:
 
             export XCODE_XCCONFIG_FILE=/tmp/tmp.xcconfig
 
-            envman add --key XCODE_XCCONFIG_FILE --value /tmp/tmp.xcconfig 
+            envman add --key XCODE_XCCONFIG_FILE --value /tmp/tmp.xcconfig
```
Comment (nbhasin2) [inline bitrise.yml:431]: @dnarcese I am seeing some extra changes in the file that are not related to v31.x branch, are these changes really required for BR? 🤔

Comment (dnarcese) [reply bitrise.yml:431]: They should all just be whitespace changes, right?

Comment (nbhasin2) [reply bitrise.yml:431]: Yeah just whitespace

Comment (dnarcese) [reply bitrise.yml:431]: Those were auto changed from Bitrise. They won't affect anything so I figured it was easier to leave them then go through and undo them all

Comment (nbhasin2) [reply bitrise.yml:431]: Oh so this was edited from BR and you just downloaded the file and are now updating it in the repo right?

Comment (dnarcese) [reply bitrise.yml:431]: yes

**Review by nbhasin2 — APPROVED** [review]
